gems/smithy/spec/fixtures/shapes/model.smithy

```diff
@@ -2,96 +2,119 @@ $version: "2"
 
 namespace smithy.ruby.tests
 
-@paginated(inputToken: "nextToken", outputToken: "nextToken", pageSize: "pageSize")
+@trait
```

Comment:
TIL about `@trait`

Reply:
Yeah. This allows us to test traits on all shapes.